packages/api/src/client/object/convertWireToOsdkObjects.ts

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import { Temporal } from "@js-temporal/polyfill";
```

Comment:
Using `Temporal` kind of sketches me out. I haven't checked the size of the polyfill (which depends on the bigint polyfill too), but its also tying us to the spec which is still in the proposal stage which reminds me of the time that typescript added decorators and got bit. No browsers claim to support this yet.

Reply:
TS added decorators at stage 1. This is stage 3 where it has been for awhile and the last thing they wanted to stabilize just closed.